_posts/theme/2019-12-23-Flora.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: theme
+title: Flora
+category: theme
+homepage: https://github.com/wnanbei/typora-theme-flora
+download: https://github.com/wnanbei/typora-theme-flora/releases/latest
+built-in: false
+date: 2019-12-23
+author: Wnanbei
+thumbnail: flora.png
+typora-root-url: ../../
+typora-copy-images-to: ../../media/theme/flora
+---
+
+# typora-theme-flora
+A theme that looks as beautiful as flora.
+
+This is a theme based on `Ursine`: [https://github.com/aCluelessDanny/typora-theme-ursine](https://github.com/aCluelessDanny/typora-theme-ursine)
```

Comment:
Please also consider use a `fork`, like https://github.com/typora/typora-theme-gallery/pull/74/files if the theme has much similarity with `Ursine `

Reply:
Sorry, this is the first time I use pull request.
Do you mean I need to write a statement on Ursine's page? Because I see the repository of `Universal JP` is not fork from `Github` theme.